community/gitlab/README.md

```diff
@@ -0,0 +1,176 @@
+# GitLab Helm Chart
+
+GitLab is a web-based Git-repository manager with wiki and issue-tracking features. This `gitlab` chart is based on [GitLab's chart](https://gitlab.com/charts/gitlab).
+
+## TL;DR;
+
+To install the chart with the release name `my-release`:
+
+```bash
+helm install --name my-release community/gitlab
+```
+
+## Installing chart
+
+### Create secrets
+
+If TLS is enabled, you will need to create certificates for `unicorn`, `minio` (if installed), and the Docker registry. Instructions on how to create these certificates using ICP's Certificate manager can be found [here](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.2/manage_applications/create_cert.html).
+
+### Image Security Policies
+
+If the cluster has image security policies enforced, the following images should be added to it.
+
+```yaml
+apiVersion: securityenforcement.admission.cloud.ibm.com/v1beta1
+kind: ImagePolicy
+metadata:
+  name: gitlab-cluster-image-policy
+  namespace: gitlab
+spec:
+  repositories:
+    - name: registry.gitlab.com/*
+```
+
+The following images may also need to be allowed depending on what subcharts you are installing as part of this chart.
+
+- gitlab-runner
+  - name: docker.io/gitlab/gitlab-runner:*
+
+- minio
+  - name: docker.io/minio/minio:*
+  - name: docker.io/minio/mc:*
+
+- postgresql
+  - name: docker.io/postgres:*
+  - name: docker.io/wrouesnel/postgres_exporter:*
+
+- redis
+  - name: docker.io/redis:*
+  - name: docker.io/oliver006/redis_exporter:*
+
+- prometheus
+  - name: docker.io/prom/prometheus:*
+  - name: docker.io/prom/alertmanager:*
+  - name: docker.io/prom/node-exporter:*
+  - name: docker.io/prom/pushgateway:*
+  - name: docker.io/jimmidyson/configmap-reload:*
+  - name: k8s.gcr.io/kube-state-metrics:*
+
+- registry
+  - name: docker.io/registry:*
+
+- operator
+  - name: gcr.io/google_containers/hyperkube:*
+
+- cert-manager
+  - name: quay.io/jetstack/cert-manager-controller:*
+
+- nginx-ingress
+  - name: quay.io/kubernetes-ingress-controller/nginx-ingress-controller:*
+  - name: k8s.gcr.io/defaultbackend:*
+
+For documentation on managing image policies refer to [Enforcing container image security](https://www.ibm.com/support/knowledgecenter/en/SSBS6K_3.1.2/manage_images/image_security.html).
+
```

Comment:
What is the minimum PSP required to deploy Gitlab?  Should be a section in the README for that. https://github.com/IBM/charts/blob/master/GUIDELINES.md#pod-security